### DISCUSSION TITLE
Bound parameters

### DISCUSSION THREAD

**Discussion by janscholten** [sfu-db/connector-x#384] [Q&A]
Hi,

I need bound parameters for security. In SQLAlchemy, I'd do 
```python
t = text("SELECT * FROM users WHERE id=:user_id")
result = connection.execute(t, user_id=12)
```
Does `connectorx` feature something like that?

**Comment by wangxiaoying** [comment]
Hi @janscholten , currently connectorx does not support this feature.

**Reply by gregglind** [reply]
What would it take to add this feature?

**Reply by UserNobody14** [reply]
Seconding, this is fairly important for secure db access
